### ISSUE TITLE
False positive WPS323 with logging configuration

### ISSUE THREAD

**Issue by fpuga** [wemake-services/wemake-python-styleguide#2638]
### What's wrong

I have a Django app and in `settings.py` I configure logging like this:

```python
LOGGING = {
    // some settings
    "formatters": {
        "app": {
            "format": (
                "%(asctime)s [%(levelname)-8s] "  // WPS323
                "(%(module)s.%(funcName)s) %(message)s" // WPS323
            ),
            "datefmt": "%Y-%m-%d %H:%M:%S", // WPS323
        }
    },
}
```

The lines marked are raising a WPS323 warning.

### How it should be

IMHO this is a false positive and `%` should be excluded from  logging config.

**EDIT**:

After posting the bug i found #1304 that is a bit similar. Maybe if this is the expected behaviour adding a note in the [documentation of](https://wemake-python-styleguide.readthedocs.io/en/latest/_modules/wemake_python_styleguide/violations/consistency.html#ModuloStringFormatViolation) would be a good idea.

### Flake8 version and plugins

wemake_python_styleguide==0.17.0

### pip information

pip==23.1.1

### OS information

Ubuntu 20.04.6 LTS

**Comment by sobolevn** [comment]
This rule is now disabled.
